### ISSUE TITLE
Something went wrong when connecting to Facebook!  Please try again.

### ISSUE THREAD

**Issue by t9fdservices** [facebookincubator/facebook-for-woocommerce#1529]
I get this error when I try to connect my website to Facebook. 
Something went wrong when connecting to Facebook!  Please try again.
![Screen Shot 2020-09-04 at 10 38 04 AM](https://user-images.githubusercontent.com/34955469/92270710-5dd80080-ee9b-11ea-9e0f-a690b32e2779.png)
![Screen Shot 2020-09-04 at 10 37 58 AM](https://user-images.githubusercontent.com/34955469/92270715-5e709700-ee9b-11ea-9e1f-ff0c61487949.png)
![Screen Shot 2020-09-04 at 10 37 49 AM](https://user-images.githubusercontent.com/34955469/92270720-5f092d80-ee9b-11ea-9178-89ea27cc98c2.png)
![Screen Shot 2020-09-04 at 10 37 45 AM](https://user-images.githubusercontent.com/34955469/92270723-5fa1c400-ee9b-11ea-91e8-fde913ec817f.png)
![Screen Shot 2020-09-04 at 10 37 40 AM](https://user-images.githubusercontent.com/34955469/92270725-603a5a80-ee9b-11ea-80b4-a25112fa3efe.png)
![Screen Shot 2020-09-04 at 10 37 35 AM](https://user-images.githubusercontent.com/34955469/92270726-60d2f100-ee9b-11ea-853b-5dbe2c0970e5.png)
![Screen Shot 2020-09-04 at 10 37 30 AM](https://user-images.githubusercontent.com/34955469/92270728-60d2f100-ee9b-11ea-8c57-afe83fb65486.png)
![Screen Shot 2020-09-04 at 10 37 26 AM](https://user-images.githubusercontent.com/34955469/92270732-616b8780-ee9b-11ea-8920-24933fe320d8.png)

``

**Comment by t9fdservices** [comment]
Images are in backwards order. but yo get what's going on. it is a issue on all of our websites. V2 will not connect but the old version will

**Comment by sandrodz** [comment]
This is a duplicate of #1521

**Comment by lgfogle** [comment]
Hey @t9fdservices,

Thanks for reaching out! I'm afraid a number of merchants have reported connection issues like the one you're experiencing - I'm so sorry about that! We are working closely with Facebook to identify and correct these issues in their connection process. 

Would you also be willing to share the **email address used for your Facebook account** and your **Business Manager ID**? You can find the Business Manager ID by going to the [Business Manager](https://business.facebook.com/), selecting the business you want to connect to, and checking the URL for the `business_id=` value. You can use a [QuickForget](www.quickforget.com) link to share this information. This will let the Facebook team look at the logs from your specific connection attempt.

Thank you!
Lindsey

P.S. @sandrodz, this is a different (though related) connection issue as what's been reported in #1521. I've followed up with you in that thread!

**Comment by t9fdservices** [comment]
Attached is the quick forget url. I believe ei got the correct information. Thank you for your help! I hope this can get resolved soon. 

https://quickforget.com/s/bfa041a22f9fd122c2b7f35d52c4f6d1f9815bc85bd026c6

**Comment by lgfogle** [comment]
Got it! Thank you so much, @t9fdservices. 

I will pass this information along to Facebook and follow up once I know more! Please don't hesitate to reach out with other questions in the meantime, and thank you for your patience.

**Comment by t9fdservices** [comment]
No problem! Thank you so much I look forward to hearing what we find out.

**Comment by Leon0824** [comment]
Try the solution below. It works for me.

- [error connect to facebool](https://wordpress.org/support/topic/error-connect-to-facebool/#post-13439968)

**Comment by sandrodz** [comment]
> Try the solution below. It works for me.
> 
> * [error connect to facebool](https://wordpress.org/support/topic/error-connect-to-facebool/#post-13439968)

Didn't work for me, I don't have any weird characters in that options value.

**Comment by Leon0824** [comment]
> > Try the solution below. It works for me.
> > 
> > * [error connect to facebool](https://wordpress.org/support/topic/error-connect-to-facebool/#post-13439968)
> 
> Didn't work for me, I don't have any weird characters in that options value.

Because your case is different with this one.

**Comment by lgfogle** [comment]
Hey @t9fdservices,

The Facebook team recently made some updates that they believe may have fixed this issue. Could you please try connecting again and let me know how it goes?

Thank you,
Lindsey

**Comment by t9fdservices** [comment]
Hey @lgfogle,

it is still doing the exact same thing. no change

**Comment by efoula** [comment]
+1

**Comment by t9fdservices** [comment]
@lgfogle This is still a issue for me. what is the solution. i still cant get my products to sync with facebook. 

I need this resolved ASAP! it is effecting my business.

Thanks

**Comment by decsu** [comment]
At step "What is Woocommerce Extension V2 allowed to do" make sure all fields are open inclusive "manage your bussiness" in this case it is open
but the default option is to be closed
he can help others with this advice even if in the case of this topoic he has all the fields open.

**Comment by PProtonzz** [comment]
Sorry for late reply.

I think we can solve this issue by 3 type.

1. Use latest Facebook for Woo-commerce plugin.

2. Don't use new Facebook account, wait at-least 1 day.

3. Below.

I often see this issue come up when there are non-ASCII characters in the page title. I took a look and noted that there are some non-ASCII characters in the title of the site. This data gets saved to Facebook in a field called wc_facebook_external_business_id and we will want to change this to see if it allows your connection to Facebook to complete successfully.

There is a method to change this within the site itself, so let’s start by trying out the steps to change this to see if it helps:

Go to https://pranavfulkari.com/ wp-admin/options.php
Search for wc_facebook_external_business_id
In the field next to it, change the data in that field to just the basic title of the page without any extra characters in place.
Scroll to the bottom of this page and click Save Changes.
In some cases, the data may not be saved due to larger databases and / or PHP settings, so after saving this, locate the wc_facebook_external_business_id field again and verify that the changes in place did save.
Once this is done, can you please try to connect to Facebook again and let me know if you get any further in the connection process?

Cheers,
Pranav Fulkari

**Comment by t9fdservices** [comment]
@PProtonzz 

For a minute there I thought this would work but nope. tried changing the wc_facebook_external_business_id several times but would never take the changes.

**Comment by devilmind81** [comment]
Hi all, 
I'm using WP 5.5.3 and I was the same issue.
I have overcame it only enabling the voice "manage your bussiness" at the step 5 (By default is disabled).

**Comment by t9fdservices** [comment]
@devilmind81 I have done the same thing but still no charge. always errors out

**Comment by diogoalmeidaribeiro** [comment]
Anyone got this to work?

**Comment by royroosterz** [comment]
Same issue here...

**Comment by devilmind81** [comment]
some updates.....after completed succesfully the wizard, only enabling the "manage your bussiness" voice at step 5, it is completed fine but after some hours my Fb account is locked suddently for souspicious activity and only a products syncronizzation was done :-O 
Please someone fix it....

**Comment by baba43** [comment]
Honestly, why are you guys not showing us any useful error information? 

The current error message is so usless and if you google it, you find hundres of entries everywhere.

This plugin is so frustrating..

**Comment by diogoalmeidaribeiro** [comment]
After hours and hours I finally got this to work. I looked the wc_facebook_external_business_id on the options table and replaced any characters after the name of the page. Initially this didn't work but after purging the cache, re-installing the plugin I got it to work.

**Comment by faolie** [comment]
> At step "What is Woocommerce Extension V2 allowed to do" make sure all fields are open inclusive "manage your bussiness" in this case it is open
> but the default option is to be closed
> he can help others with this advice even if in the case of this topoic he has all the fields open.

This worked for me. Thanks!

**Comment by t9fdservices** [comment]
@baba43 - what do you want to see? I have no idea where to get other log information regarding the issue. Geeze

@diogoalmeidaribeiro - IDK what you did I have tried all of this and to no avail I CANNOT Get it to work. im so over it. screw Facebook

@faolie - No change for me I have done that a million times. its screwed.

**Comment by t9fdservices** [comment]
so I fixed the issue for myself. finally!! I went to my business.facebook.com and under Business Settings > Apps I removed WooCommerce Extension V2. Tried it again and it worked. Magically. aha

@devilmind81 @diogoalmeidaribeiro @royroosterz

**Comment by devilmind81** [comment]
@t9fdservices, I'm happy for you,  I'm still waiting that FB will unlock my account :(

**Comment by jpbacilio** [comment]
> At step "What is Woocommerce Extension V2 allowed to do" make sure all fields are open inclusive "manage your bussiness" in this case it is open
> but the default option is to be closed
> he can help others with this advice even if in the case of this topoic he has all the fields open.

This also worked for me! (Nov 13, 2020)

**Comment by thiagomorello** [comment]
having the same error here, can't connect my account.
cannot find wc_facebook_external_business_id on options.php

do anybody have a solution?

**Comment by gioramies** [comment]
> At step "What is Woocommerce Extension V2 allowed to do" make sure all fields are open inclusive "manage your bussiness" in this case it is open
> but the default option is to be closed
> he can help others with this advice even if in the case of this topoic he has all the fields open.

This worked for me. Thank you.

**Comment by racemsmaoui** [comment]
> some updates.....after completed succesfully the wizard, only enabling the "manage your bussiness" voice at step 5, it is completed fine but after some hours my Fb account is locked suddently for souspicious activity and only a products syncronizzation was done :-O
> Please someone fix it....

that's work for me also, thanks

**Comment by alejoarosemena** [comment]
> At step "What is Woocommerce Extension V2 allowed to do" make sure all fields are open inclusive "manage your bussiness" in this case it is open
> but the default option is to be closed
> he can help others with this advice even if in the case of this topoic he has all the fields open.

This was it!!! THANK YOUUU!!!!

**Comment by Kesiunas** [comment]
> At step "What is Woocommerce Extension V2 allowed to do" make sure all fields are open inclusive "manage your bussiness" in this case it is open
> but the default option is to be closed
> he can help others with this advice even if in the case of this topoic he has all the fields open.

worked for me! Thank you!

**Comment by mrabro** [comment]
> Try the solution below. It works for me.
> 
> * [error connect to facebool](https://wordpress.org/support/topic/error-connect-to-facebool/#post-13439968)

Worked for me!

**Comment by lucagrandicelli** [comment]
This one worked for me:
At step “What is Woocommerce Extension V2 allowed to do” make sure all fields are open. Facebook recently updated their settings to toggle “Manage your business” permission to Off by default.

**Comment by LAURENTIU2021** [comment]
> so I fixed the issue for myself. finally!! I went to my business.facebook.com and under Business Settings > Apps I removed WooCommerce Extension V2. Tried it again and it worked. Magically. aha
> 
> @devilmind81 @diogoalmeidaribeiro @royroosterz

I have same issue,but i tryed to do like you but not working! I also opened all fields manage your bussiness....same error Something went wrong when connecting to Facebook! Please try again.
Is a text to look back to set up bussiness if i open the field manage your bussiness.
Did somebody know something about this?
Please tell me ASAP...since a week ago i have this issue...

**Comment by LAURENTIU2021** [comment]
> Sorry for late reply.
> 
> I think we can solve this issue by 3 type.
> 
> 1. Use latest Facebook for Woo-commerce plugin.
> 2. Don't use new Facebook account, wait at-least 1 day.
> 3. Below.
> 
> I often see this issue come up when there are non-ASCII characters in the page title. I took a look and noted that there are some non-ASCII characters in the title of the site. This data gets saved to Facebook in a field called wc_facebook_external_business_id and we will want to change this to see if it allows your connection to Facebook to complete successfully.
> 
> There is a method to change this within the site itself, so let’s start by trying out the steps to change this to see if it helps:
> 
> Go to https://pranavfulkari.com/ wp-admin/options.php
> Search for wc_facebook_external_business_id
> In the field next to it, change the data in that field to just the basic title of the page without any extra characters in place.
> Scroll to the bottom of this page and click Save Changes.
> In some cases, the data may not be saved due to larger databases and / or PHP settings, so after saving this, locate the wc_facebook_external_business_id field again and verify that the changes in place did save.
> Once this is done, can you please try to connect to Facebook again and let me know if you get any further in the connection process?
> 
> Cheers,
> Pranav Fulkari

is not working for me....still same error.

**Comment by mw-kanj** [comment]
> Sorry for late reply.
> 
> I think we can solve this issue by 3 type.
> 
> 1. Use latest Facebook for Woo-commerce plugin.
> 2. Don't use new Facebook account, wait at-least 1 day.
> 3. Below.
> 
> I often see this issue come up when there are non-ASCII characters in the page title. I took a look and noted that there are some non-ASCII characters in the title of the site. This data gets saved to Facebook in a field called wc_facebook_external_business_id and we will want to change this to see if it allows your connection to Facebook to complete successfully.
> 
> There is a method to change this within the site itself, so let’s start by trying out the steps to change this to see if it helps:
> 
> Go to https://pranavfulkari.com/ wp-admin/options.php
> Search for wc_facebook_external_business_id
> In the field next to it, change the data in that field to just the basic title of the page without any extra characters in place.
> Scroll to the bottom of this page and click Save Changes.
> In some cases, the data may not be saved due to larger databases and / or PHP settings, so after saving this, locate the wc_facebook_external_business_id field again and verify that the changes in place did save.
> Once this is done, can you please try to connect to Facebook again and let me know if you get any further in the connection process?
> 
> Cheers,
> Pranav Fulkari

Thank you so much. It worked after changing the title. I greatly appreciate your help!

**Comment by ijjimem** [comment]
> > Sorry for late reply.
> > I think we can solve this issue by 3 type.
> > 
> > 1. Use latest Facebook for Woo-commerce plugin.
> > 2. Don't use new Facebook account, wait at-least 1 day.
> > 3. Below.
> > 
> > I often see this issue come up when there are non-ASCII characters in the page title. I took a look and noted that there are some non-ASCII characters in the title of the site. This data gets saved to Facebook in a field called wc_facebook_external_business_id and we will want to change this to see if it allows your connection to Facebook to complete successfully.
> > There is a method to change this within the site itself, so let’s start by trying out the steps to change this to see if it helps:
> > Go to https://pranavfulkari.com/ wp-admin/options.php
> > Search for wc_facebook_external_business_id
> > In the field next to it, change the data in that field to just the basic title of the page without any extra characters in place.
> > Scroll to the bottom of this page and click Save Changes.
> > In some cases, the data may not be saved due to larger databases and / or PHP settings, so after saving this, locate the wc_facebook_external_business_id field again and verify that the changes in place did save.
> > Once this is done, can you please try to connect to Facebook again and let me know if you get any further in the connection process?
> > Cheers,
> > Pranav Fulkari
> 
> is not working for me....still same error.

Helped me too !!! THANKS

**Comment by NohB** [comment]
Had this problem and just wanted to let you know how I solved it. 

Had connected my Instagram account via bussiness.facebook.com to my page but for some reason found out later that my Instagram account was not connected to my page when I looked at it via facebook. Once I connected it my facebook.com all went well. The facebook plugin for woocmerce did pick up the Instagram and facebook page connection but for some reason the link has to be established by both facebook business and through facebook.com. As weird as it sounds it worked for me.

Hope it helps.
